code/modules/mob/living/carbon/human/human.dm

```diff
@@ -841,6 +842,37 @@
 			to_chat(usr, "Failed to apply DNA Infusion to [src].")
 			log_admin("[key_name(usr)] failed to apply a DNA Infusion to [key_name(src)].")
 
+	if(href_list[VV_HK_TURN_INTO_MMI])
+		if(!check_rights(R_DEBUG))
+			return
+
+		var/result = input(usr, "This will delete the mob, are you sure?", "Turn into MMI") in list("Yes", "No")
+		if(result != "Yes")
+			return
+
+		if(!ishuman(src))
+			to_chat(usr, "This can only be done to humanoid mobs.")
+			return
+
+
+		var/obj/item/organ/internal/brain/target_brain = src.get_organ_slot(ORGAN_SLOT_BRAIN)
+
+		if(!target_brain)
+			to_chat(usr, "This mob has no brain to insert into an MMI.")
+			return
+
+		var/obj/item/mmi/new_mmi = new(src.loc)
```

Comment:
```suggestion
		var/obj/item/mmi/new_mmi = new(get_turf(src))
```

safety? can ignore this if you think it's too nitty.

Reply:
The old was intentional cuz it would allow for changing a mob in a thing to an MMI and it remaining in the thing, but it probably is safer to just drop it on the turf and admins that can use this proc can probably do anything else manually anyway (SDQL2 comes to mind).

Reply:
Changed.